test/utils/constants.ts

```diff
@@ -1,8 +0,0 @@
-import { BigNumber } from 'ethers';
```

Comment:
Don't you think all the other variables are useful??

Reply:
They are all in ethersprojects constants file 😋 : https://github.com/ethers-io/ethers.js/blob/master/packages/constants/src.ts/index.ts

Reply:
I'll wait for your comment before merge @0xGorilla

Reply:
Great!! TY